en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-errors.md

```diff
@@ -1,2 +1,69 @@
-!!! Note
-    Content to be added. WIP.
\ No newline at end of file
+# Error Page
+![error page full]({{base_path}}/assets/img/observe/error/error-page-full.png)
+
+The error page displays analytics stats related to erroneous API calls that happened within the API management deployment. 
+Page will include error categorization based on the error type and then further drills down into sub types. 
+You can use this page as the starting point for debugging any API errors.
+See the topics given below to get details of the available widgets.
+
+## Common Controls
+![common controls]({{base_path}}/assets/img/observe/error/common-controls.png)
+
+The Error page offers you the three additional common controls listed below.
+### API Selector
+Use the **API Selector** to select the APIs for which you want to view analytics. By default, this is set to **All**. You can select one or multiple APIs from this selector and view the aggregated result.
+
+### Category/Status Code Toggle
+Use this toggle to switch between error category view or status code view. When you are using **Category** mode, you will see errors categorized by predefined types. 
+In the **Status Code** mode, you will observe errors categorized by the client-side status code. Category view is more suitable for business analytics purposes while 
+status code mode is more suitable for troubleshooting and finding faulty APIs. Depending on this toggle other common control and chart layout will change.
+
+### Category Selector
+**Category Selector** is enabled only when you have selected **Category** toggle explained above. Using this selector you can select one or all of below error categories.
+- Authentication: Any kind of authentication error will fall into this category including expired/missing/invalid credentials.
+- Target Connectivity: Any kind of API backend errors will fall into this category including connection time-outs/backend error(4xx ,5xx status codes)
```

Comment:
"Target Connectivity: Any kind of API back-end error falls into this category including connection time-outs/backend error(4xx ,5xx status codes)."

I think the last statement 'back-end error (4xx, 5xx status codes)' is not very clear. Are we trying to say 'connection timeouts and back-end errors with error codes ...'?

Reply:
what i try to mean is any error happening within the backend service we are using. Generally when this happens that backend will return a 5xx code to api gateway